requirements.txt

```diff
@@ -1,8 +1,2 @@
 setuptools
```

Comment:
setuptools usually shouldn't be added as a dependency, unless the package depends on features from it at runtime. That's not the case here, so I would suggest to remove it.